packages/js-example-app/src/index.js

```diff
@@ -31,27 +32,39 @@ toggleLoadingButton.addEventListener('click', () => {
 });
 
 toggleDisabledState.addEventListener('click', () => {
-  const currentState = blrButton.getAttribute('disabled');
+  const currentState = blrButtonText.getAttribute('disabled');
 
   if (currentState) {
-    blrButton.removeAttribute('disabled');
-    addLog('Set button text disabled state to false');
+    blrButtonText.removeAttribute('disabled');
+    addLog('Set text button disabled state to false');
   } else {
-    blrButton.setAttribute('disabled', 'true');
-    addLog('Set button text loading state to true');
+    blrButtonText.setAttribute('disabled', 'true');
+    addLog('Set text button loading state to true');
   }
 });
 
-blrButton.addEventListener('blrClick', () => {
-  addLog('blr-button-text clicked');
+blrButtonText.addEventListener('blrClick', () => {
+  addLog('blr-text-button clicked');
+});
+
+blrButtonText.addEventListener('blrFocus', () => {
+  addLog('blr-text-button focused');
+});
+
+blrButtonText.addEventListener('blrBlur', () => {
+  addLog('blr-text-button blurred');
+});
+
+blrButtonIcon.addEventListener('blrClick', () => {
+  addLog('blr-icon-button clicked');
 });
 
-blrButton.addEventListener('blrFocus', () => {
-  addLog('blr-button-text focused');
+blrButtonIcon.addEventListener('blrFocus', () => {
+  addLog('blr-icon-button focused');
```

Comment:
should be blr-button-icon for consistency.

Reply:
Ok, fair enough. That has been resolved